compose/ui/ui/api/desktop/ui.api

```diff
@@ -2002,8 +2002,6 @@ public final class androidx/compose/ui/input/pointer/PointerEvent_skikoKt {
 	public static final fun isShiftPressed-5xRPYO0 (I)Z
 	public static final fun isSymPressed-5xRPYO0 (I)Z
 	public static final fun isTertiaryPressed-aHzCx-E (I)Z
-	public static final fun toCompose (Lorg/jetbrains/skiko/SkikoPointerDevice;)I
-	public static final fun toCompose (Lorg/jetbrains/skiko/SkikoPointerEventKind;)I
```

Comment:
It's a binary compatibility breaking change. It requires
1. Discussion with @igordmn 
2. Deprecate it first

I know it probably isn't supposed to be public at all, but it's a blocker here

Reply:
@igordmn my fellow composers but this is literally entities that does not exist in the branch @MatkovIvan is working at - there's no way this method will be needed because there's no such entities this methods are supposed to convert from. Also, this was indeed strange that those methods are considered part of API.

Reply:
It contains Skiko type, so it is safe to remove associated functions.

If a type itself isn't backward compatible (because it is in Skiko), an associated Compose function don't support backward compatiblity for type renaming/removing.